### PR TITLE
Show app version.

### PR DESCRIPTION
**Pull request by rpiaggio**
Shows:

* On production: `YYYYMMDD-HASH(7)`
* On staging: `YYYYMMDD-HASH(7)-STG`
* On development: `YYYYMMDD-HHMMSS-DEV`

Couldn't find a place in the drop down menu that convinced me, so I placed it on an "Info" icon, but I'm open to other suggestions.

image

image

Clicking on the icon on the right copies the version to the clipboard.

### CODE REVIEW

File: common/src/main/resources/less/style.less [hunk 1]
```diff
@@ -1268,9 +1268,29 @@ i.icon.user-astronaut:before { content: "\f4fb"; }
...
 }
+
+i.icons.info .corner.icon {
+  font-size: 0.65em;
+  text-shadow: none;
+}
+
+.ui.popup.discrete-popup {
+  padding: 5px 10px;
+  box-shadow: none;
```
Comment (cquiroz) [inline common/src/main/resources/less/style.less:1285]: We should maybe remove shadows for all popup, they don't look nice in dark

Comment (cquiroz) [reply common/src/main/resources/less/style.less:1285]: Maybe in an about/help box triggered from the menu?

File: explore/src/main/scala/explore/TopBar.scala [hunk 2]
```diff
@@ -56,7 +70,34 @@ object TopBar {
...
+                    )
+                  )(
+                    PopupContent(
+                      <.span(ExploreStyles.Version)(
+                        s"Version: ${appCtx.version}",
+                        CopyToClipboard(
+                          text = appCtx.version.value,
+                          onCopy = (_, copied) =>
+                            $.setState(State(copied)) >>
+                              $.setState(State(false)).delayMs(1500).toCallback
```
Comment (toddburnside) [inline explore/src/main/scala/explore/TopBar.scala:91]: The temporary check mark is a nice touch.

Comment (cquiroz) [reply explore/src/main/scala/explore/TopBar.scala:91]: Yeah, much better than the toast we have on the seqexec

**Comment by rpiaggio** [comment]
Ok, I added an `About` option to the dropdown menu:

image


Which now shows a modal, where I shamelessly copied the Explore logo and animation that @cquiroz made for the login screen:

image

**Comment by cquiroz** [comment]
Nice, we should show the names and emails of ever developer ( but me :D)

**Comment by toddburnside** [comment]
> Nice, we should show the names and emails of ever developer ( but me :D)

Good idea. My email address is haha@getlost.com

File: common/src/main/scala/explore/components/About.scala [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+case class About(trigger: VdomNode, content: VdomNode) extends ReactProps[About](About.component)
+
+object About {
+  type Props = About
+
+  protected val component = ScalaComponent
+    .builder[Props]
+    .render_P { props =>
+      Modal(
```
Comment (cquiroz) [inline common/src/main/scala/explore/components/About.scala:22]: Will it close on `ESC`?

**Review by cquiroz — APPROVED** [review]

**Review by toddburnside — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/About.scala (hunk 3 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/components/About.scala:22]: Yes!

**Pull request merged**